packages/config/config/devices/0x027a/zen32.json

```diff
@@ -519,6 +519,46 @@
 					"value": 1
 				}
 			]
+		},
+		{
+			"#": "22",
+			"$if": "firmwareVersion >= 10.0",
+			"label": "Programming through Relay Button",
+			"description": "Controls whether certain features (inclusion, exclusion, LED indicator modes) can be programmed through the relay button. Disabling this frees up the triple tap action for scene control. Factory reset will always work regardless of this setting.",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Enable",
+					"value": 0
+				},
+				{
+					"label": "Disable",
+					"value": 1
+				}
+			]
+		},
```

Comment:
This seems to be the same as parameter 17 on the ZEN71. Please update the definition here to be the same. Ideally this should probably be moved into the `zooz_template.json` since they're using it on more than one device now.

Reply:
Good point! It's already in the templates: https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x027a/templates/zooz_template.json#L255= (used by ZEN72). So I'll fix both this and ZEN71 not using the template.

Reply:
Do we care that ZEN32 technically doesn't have a paddle but a (relay) button instead?

Reply:
@blhoward2 what do you think?

Reply:
@kaechele see below for the answer :)

Reply:
Thanks. Found it. I will update this PR with the proposed changes.